### DISCUSSION TITLE
Minor updates to SAFE Stack template - easy way of incorporating these with an existing project?

### DISCUSSION THREAD

**Discussion by ciphertools** [SAFE-Stack/SAFE-template#611] [Q&A] [answered]
When a major release of the SAFE Stack template is issued, I tend to rebuild my existing projects with the new template from scratch.  A minor update to the SAFE Stack template to 5.0.5 came out last week, and I am wondering if there is an easy way to incorporate this new version into existing projects that I have already updated to SAFE Stack v5, without having to rebuild these projects from scratch.

**Comment by Larocceau** [comment, marked as answer]
Hi @ciphertools! We recently started using Meta packages, that should help you a lot! These meta packages define all dependencies in one go. That means that, if a new SAFE version comes out, you can simply update your dependencies on SAFE.Server and SAFE.Client. Of course on major releases this also leads to some changes in your application.

**Reply by ciphertools** [reply]
Thank you, and thank you for SAFE Stack - I love working with it!
